### DISCUSSION TITLE
Autopilot Manager support for Auto Scaling

### DISCUSSION THREAD

**Discussion by neilpaton** [okieselbach/Autopilot-Manager#19] [Q&A] [answered]
Hi Oliver
Does the Autopilot Manager web app support autoscaling?
We have an App Service Plan with the autoscale option enabled. This plan is shared by a number of our web apps including Autopilot Manager. When the instance count went above 1, we noticed that the Autopilot Manager web app didn't run as expected.
I just wanted to check if this is expected.

Thanks
Neil

**Comment by okieselbach** [comment, marked as answer]
Sorry for my delay, unfortunately, I never coded it with scaling in mind. I use a memory cache for some states, so I guess this will not work correctly. So yeah expected.
